CMakeLists.txt

```diff
@@ -212,6 +212,46 @@ set (
     "download_handlers"
     CACHE STRING "Sub folder for download handler extensions.")
 
+set (
+    ADUC_DOWNLOAD_HANDLER_EXTENSION_DIR
+    "${ADUC_EXTENSIONS_FOLDER}/${ADUC_EXTENSIONS_SUBDIR_DOWNLOAD_HANDLERS}"
+    CACHE STRING "Path for the download handler extensions")
+
+if (ADUC_BUILD_UNIT_TESTS)
+    set (
+        EIS_DATA_PATH
+        "/tmp/eis_conn_info.txt"
```

Comment:
release artifacts are often built with -u so this would have the wrong path in production binaries.  Instead, can have 'testdata' folder under ut folder and the copy_test_data CMake macro will be run that will copy it under a deterministic test directory (under tmp):

https://github.com/Azure/iot-hub-device-update/blob/99ab06fbdf9dc91f4015eb83fc2f4a2bf977346c/cmake/aduc_helpers.cmake#L51